launch.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
+
+# getopts "hfr" args;
```

Comment:
I struggled a bit to have a `-h` option that would show more info. didn't want to spend too much time on that. lmk if you know how to do that, otherwise let's forget about it.

Reply:
while getopts ":h:" opt; do
  case $opt in
    a)
      echo "-h was triggered, Parameter: $OPTARG" >&2
      ;;
    \?)
      echo "Invalid option: -$OPTARG" >&2
      exit 1
      ;;
    :)
      echo "Option -$OPTARG requires an argument." >&2
      exit 1
      ;;
  esac
done

Reply:
That doesn't work
```bash
?)
echo "Invalid option: -$OPTARG" >&2
```

Will break with -f or -r.
Mreging as I don't want to lose time on this, it's just for us.